### ISSUE TITLE
A11Y: Skin Tone Picker should be accessible

### ISSUE THREAD

**Issue by comp615** [missive/emoji-mart#220]
The Skin tone swatch in the preview bar should be accessible for both ARIA and keyboard navigation. This could be presented as a select menu or similar component.

**Comment by nolanlawson** [comment]
Starting to look into this. Seems the best design may be:

```html
<button aria-haspopup="true" aria-expanded={!!opened}>
</button>
```

The challenge here is that the options are currently _inside_ the button (which is a `<div>` currently). I think we'll have to change it so the HTML is outside of it, with `role="menu"`
